test/CheckRightsService.Broker.UnitTests/CheckRightsService.Broker.UnitTests.csproj

```diff
@@ -2,4 +2,9 @@
 
   <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\CheckRightsService.Broker\CheckRightsService.Broker.csproj" />
```

Comment:
Probably, you should not write it here, as I know, we have targets in project, there are all packages. I may be wrong

Reply:
If I don't do it here, I won't have any entities from Broker, but I need to tests them.